src/drivers/batt_smbus/batt_smbus.cpp

```diff
@@ -310,59 +305,55 @@ int BATT_SMBUS::dataflash_write(uint16_t address, void *data, const unsigned len
 	memcpy(&tx_buf[2], data, length);
 
 	// code (1), byte_count (1), addr(2), data(32) + pec
-	int result = _interface->block_write(code, tx_buf, length + 2, false);
+	int ret = _interface->block_write(code, tx_buf, length + 2, false);
 
-	return result;
+	return ret;
 }
 
 int BATT_SMBUS::get_startup_info()
 {
-	int result = 0;
-
-	// The name field is 21 characters, add one for null terminator.
-	const unsigned name_length = 22;
+	int ret = PX4_OK;
 
 	// Read battery threshold params on startup.
-	param_get(param_find("BAT_CRIT_THR"), &_crit_thr);
-	param_get(param_find("BAT_LOW_THR"), &_low_thr);
-	param_get(param_find("BAT_EMERGEN_THR"), &_emergency_thr);
-	param_get(param_find("BAT_C_MULT"), &_c_mult);
-
-	// Try and get battery SBS info.
-	if (_manufacturer_name == nullptr) {
-		char man_name[name_length] = {};
-		result = manufacturer_name((uint8_t *)man_name, sizeof(man_name));
-
-		if (result != PX4_OK) {
-			PX4_DEBUG("Failed to get manufacturer name");
-			return PX4_ERROR;
-		}
+	param_get((param_t)(px4::params::BAT_CRIT_THR), &_crit_thr);
+	param_get((param_t)(px4::params::BAT_LOW_THR), &_low_thr);
+	param_get((param_t)(px4::params::BAT_EMERGEN_THR), &_emergency_thr);
+	param_get((param_t)(px4::params::BAT_C_MULT), &_c_mult);
 
-		_manufacturer_name = new char[sizeof(man_name)];
-	}
+	int32_t cell_count_param;
```

Comment:
```suggestion
	int32_t cell_count_param = 0;
```

Reply:
@dakejahl - thanks, fixed
again, rebased for clearer log.